posts/urls.py

```diff
@@ -1,11 +1,12 @@
 from django.urls import path
-from .views import PostListView, PostDetailView
+from .views import PostListView, PostDetailView, CommentCreateView
 from . import views
 
 
 urlpatterns = [
     path('', PostListView.as_view(), name='posts-feed'),
-    path('post/<int:pk>/', PostDetailView.as_view(), name='posts-detail'),
+    path('post/<int:pk>/', PostDetailView.as_view(), name='post-detail'),
```

Comment:
I think it shouldn't be a part of the PR

Reply:
I am just changing the name of the URL to `post-detail` because every detail belongs to different post and not to all posts :)

Reply:
Added this change to the PR description 👌🏼